contracts/LiquidityPool.sol

```diff
@@ -268,6 +285,7 @@ contract LiquidityPool is Pausable, ReentrancyGuard, PiAdmin {
         _timestamps[msg.sender] = uint40(block.timestamp);
 
         asset.safeTransfer(msg.sender, _amount);
+        debtSettler.addBorrower(msg.sender);
```

Comment:
```suggestion
        if (_withSettler()) debtSettler.addBorrower(msg.sender);
```
We could not use settler for "not auto repay" pools =) just in case